packages/apps/plugins/plugin-stack/src/StackPlugin.tsx

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { definePlugin, PluginDefinition } from '@dxos/react-surface';
+
+import { StackMain } from './components';
+import { isStack } from './props';
+import translations from './translations';
+
+export const StackPlugin: PluginDefinition = definePlugin({
+  meta: {
+    id: 'dxos:stack',
```

Comment:
need to start to think about GUIDs (which we've previously called DXNs). e.g., `dxos.org/plugin/stack`

Reply:
@richburdon I have a todo for this somewhere to revisit DXNs. The format of this is specifically the old DXN format but not well-defined. It's been on my mind :+1: